test/external/external_test_optim.py

```diff
@@ -84,5 +111,13 @@ def test_multistep_lars(self): self._test_lamb(10, {'lr': 0.001}, 1e-5, 0)
   def test_multistep_lars_high_lr(self): self._test_lamb(10, {'lr': 10}, 1e-5, 3e-4)
   def test_lars_skip_list(self): self._test_lars(1, {'lr': 0.01, 'skip_list': True}, 1e-5, 0)
 
+  def test_lars_polylr(self):
+    self._test_lars_polylr(10, {'lr': 1.0}, {
```

Comment:
can you add one more with bigger steps? maybe the one that resnet will use

Reply:
bigger on what dimension?

Reply:
sorry i meant steps, just one with different steps and warmup

Reply:
Hm, seems like it can run through 45 * 2053 steps pretty fast; added